add_to_build_results/monodistribution/bin/cli_x86

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. $(dirname $0)/mono-env
```

Comment:
Spoke with @joncham , we are going to let this go for now.  I was just adding this to help @lucasmeijer